recommenders/datasets/movielens.py

```diff
@@ -154,14 +149,17 @@ def load_pandas_df(
     To load movie information only, you can use load_item_df function.
 
     Args:
-        size (str): Size of the data to load. One of ("100k", "1m", "10m", "20m").
-        header (list or tuple or None): Rating dataset header.
-        local_cache_path (str): Path (directory or a zip file) to cache the downloaded zip file.
+        size (str): Size of the data to load. One of ("100k", "1m", "10m", "20m", "mock100").
+        header* (list or tuple or None): Rating dataset header.
```

Comment:
What is the meaning of * here? Is it described in the Google style?

Reply:
I intended to use it like a footnote notation for function arguments that are ignored when a mock data size is specified. I can remove it and replace with a explicit statement.

Reply:
Done